sensor/common/sensor/central_communication_impl.go

```diff
@@ -368,7 +368,7 @@ func (s *centralCommunicationImpl) initialPolicySync(ctx context.Context,
 	if err != nil {
 		return errors.Wrap(err, "receiving initial baselines")
 	}
-	if err := detector.ProcessMessage(msg); err != nil {
+	if err := detector.ProcessMessage(context.TODO(), msg); err != nil {
```

Comment:
Would it make sense to pass on the function parameter context here ?

Reply:
Yes, it makes total sense. Here it is: [db41b4f](https://github.com/stackrox/stackrox/pull/16064/commits/db41b4f08bc0f48cda2dad920e296b21ea22ef76)